doc/user/nhrpd.rst

```diff
@@ -124,9 +124,9 @@ nhrpd connects to strongSwan using VICI protocol based on UNIX socket
 (hardcoded now as /var/run/charon.vici).
 
 strongSwan currently needs few patches applied. Please check out the
-`http://git.alpinelinux.org/cgit/user/tteras/strongswan/log/?h=tteras-release,release <http://git.alpinelinux.org/cgit/user/tteras/strongswan/log/?h=tteras-release,release>`_
+`http://git.alpinelinux.org/user/tteras/strongswan/log/?h=tteras-release,release <https://git.alpinelinux.org/user/tteras/strongswan/log/?h=tteras-release>`
```

Comment:
This won't render as a link, you have to put a `_` at the end. However, this syntax isn't necessary in the first place because Sphinx will recognize hyperlinks and format them appropriately with no special markup required.